### ISSUE TITLE
Support lambda role from parameter and/or environment variables on init

### ISSUE THREAD

**Issue by rv-rsouza** [apex/apex#855]
A good addition to this awesome tool would be an option to provide the lambda role on init to avoid errors on role creation when the profile has no permissions to do so.

I had to create the boilerplate manually to be able to use apex in my projects because the company has a very restrictive access permissions policy (and this is a good thing) just because init always require the role creation.

**Comment by haidaraM** [comment]
Yes this will be a nice feature to have. Looks live the project is not actively maintained for the moment (https://github.com/apex/apex/issues/894)
